src/main/java/org/bitbucket/nanojava/CliOptions.java

```diff
@@ -0,0 +1,57 @@
+/* Copyright (C) 2011-2020  Egon Willighagen <egonw@users.sf.net>, Ammar Ammar <ammar257ammar@gmail.com>
```

Comment:
This is a new file and I did not contribute, so should not be listed as copyright owner.

Reply:
Done